src/App.vue

```diff
@@ -71,11 +71,21 @@ function sendNoteOn(frequency: number, rawAttack: number) {
   return off
 }
 
-function midiNoteOn(index: number, rawAttack?: number) {
+function midiNoteOn(index: number, rawAttack?: number, channel?: number) {
+
+  // in multichannel-to-equave mode calculate an offset based on the incoming channel
+  if (midi.multichannelToEquave && channel !== undefined) {
+    let offset =  mmod(channel - midi.multichannelCenter + midi.multichannelEquavesDown, midi.multichannelNumEquaves) - midi.multichannelEquavesDown
```

Comment:
`const offset; index += offset * scale.scale.size;` Might be nicer (prefer `const` over `let` when possible).